src/ExportMap.js

```diff
@@ -344,14 +345,43 @@ ExportMap.parse = function (path, content, context) {
   var m = new ExportMap(path)
 
   try {
-    var ast = parse(path, content, context)
+    var { ast, visitorKeys } = parse(path, content, context)
   } catch (err) {
-    log('parse error:', path, err)
     m.errors.push(err)
     return m // can't continue
   }
 
-  if (!unambiguous.isModule(ast)) return null
+  let hasDynamicImports = false
+
+  visit(ast, visitorKeys, {
+    CallExpression(node) {
```

Comment:
It's `ImportExpression` in the latest `@babel/parser` which is a dependency of `babel-eslint`
https://github.com/babel/babel/pull/10828